base/Dockerfile

```diff
@@ -100,6 +100,18 @@ RUN apk add --no-cache \
     && apk del .build-dependencies \
     && rm -rf /usr/src/arp-scan
 
+# PicoTTS - it has no specific version - unfortunately we'll have to use master directly

```

Comment:
We want reproducible builds, please lock it down to commit hash (as an alternative to version tags).

Reply:
Added commit hash as argument